File: server/cmd/trip/pkg/mq/model/PayInfo.go
package model

import "github.com/CyanAsterisk/FreeCar/server/shared/id"

type PayInfo struct {
	AccountId id.AccountID `json:"account_id"`
	TripId    id.TripID    `json:"trip_id"`
	FeeCent   int32        `json:"fee_cent"`
}

File: server/cmd/trade/src/main/java/org/lanlance/freecartrade/model/PayInfo.java
package org.lanlance.freecartrade.model;

import lombok.AllArgsConstructor;
import lombok.Data;

import com.fasterxml.jackson.annotation.JsonProperty;

@Data
@AllArgsConstructor
public class PayInfo {
    @JsonProperty("account_id")
    private String AccountID;

    @JsonProperty("trip_id")
    private String TripID;

    @JsonProperty("fee_cent")
    private Integer FeeCent;
}